src/utils/cliOptionParsing.ts

```diff
@@ -33,10 +37,39 @@ export function parsePassOrder(
     }
     const [key, nextPass] = foundPass;
     passesInOrder.push(nextPass);
+    keyPassesInOrder.push(key);
     if (key === until) break;
     remainingOrder = remainingOrder.slice(key.length);
   }
 
+  passesInOrder.forEach((element, index) => {
+    const prerequites = element._getPassPrerequites();
+    const prerequitesKeys = [...keyPassesInOrder].slice(0, index);
```

Comment:
I think we should rename this EarlierPassKeys, since these are the passes that are occurring before. Prerequisite means that it is needed before the current pass.